### PR TITLE
Improve company-tooltip-minimum docs

### PR DESCRIPTION
**Pull request by yugaego**
The suggested texts should better reflect what `company-tooltip-minimum` option influences.

### CODE REVIEW

File: company.el [hunk 1]
```diff
@@ -266,8 +266,9 @@ The visualized data is stored in `company-prefix', `company-candidates',
   :type 'integer)
 
 (defcustom company-tooltip-minimum 6
-  "The minimum height of the tooltip.
-If this many lines are not available, prefer to display the tooltip above."
+  "Ensure visibility of this number of candidates.
+When that many lines are not available between the point and bottom of the
```
Comment (dgutov) [inline company.el:270]: In Emacs lingo, we usually use the term "point" without the article.

So: "between point and ..." and "above point".

Some examples here: https://www.gnu.org/software/emacs/manual/html_node/emacs/Point.html

Comment (yugaego) [reply company.el]: Removed. Please see now.

Comment (dgutov) [reply company.el]: Could you also fix that in the manual?

Comment (yugaego) [reply company.el]: Argh! Yes, sure. 👍🏻

Comment (yugaego) [reply company.el]: And now done. Let's try again 😉

**Comment by dgutov** [comment]
Thanks!

**Pull request merged**